src/Controller/DataController.java

```diff
@@ -388,6 +378,8 @@ static private HubFile processNewHubFile(NodeList nList) throws Exception
                                     }
 
                                     break;
+                            default:
```

Comment:
What is the purpose of adding this default clause? Should it do something (like throw an exception or log an error)?